changelog.html

```diff
@@ -1,11 +1,44 @@
 <div class="title" i18n="defaultChangelogHead"></div>
 
+<span>2020.04.27 - 10.7.0 - BetaFlight</span>
+<ul>
+    <li>Features:
+        <ul>
+            <li>improved target selection in the firmware flasher tab</li>
+            <li>improved PID tuning and the tuning sliders</li>
+            <li>added a button to bind SPI RX / newer Spektrum RX from within configurator</li>
+            <li>added warning messages about potential configuration issues</li>
+        </ul>
+    </li>
+    <li>Fixes:
+        <ul>
+            <li>fixed the flashing progress bar</li>
+            <li>fixed the PID tab layout</li>
+            <li>various other UI fixes</li>
+        </ul>
+    </li>
+    <li>New GUI support for the following firmware features:
+        <ul>
+            <li>added rates type selection</li>
+            <li>added support for more settings in PID profiles</li>
+            <li>added support for more settings for GPS</li>
+        </ul>
+    </li>
+    <li>New Languages:
+        <ul>
+            <li>Euskera (Basque)</li>
+            <li>Portugu&ecirc;s Brasileiro (Brasilian Portugese)</li>
+            <li>polski (Polish)</li>
```

Comment:
I think Hungarian need to be added ;)

Reply:
And Chinese Traditional too now 🤪

Reply:
![image](https://user-images.githubusercontent.com/4742747/80322112-d5a37800-8876-11ea-8bfb-892a005b63b4.png)